proxy/src/http/websocket.rs

```diff
@@ -201,7 +201,12 @@ async fn ws_handler(
             .await;
         let status_code = match result {
             Ok(_) => StatusCode::OK,
-            Err(_) => StatusCode::BAD_REQUEST,
+            Err(ref e) => {
+                if cfg!(debug_assertions) {
+                    error!("error in sql-over-http: {}, backtrace: {}", e, e.backtrace());
+                }
+                StatusCode::BAD_REQUEST
+            }
```

Comment:
Can we revert this change

Reply:
I thought it would be more helpful for debugging purpose but I'm also fine with reverting it :)

Reply:
fixed in https://github.com/neondatabase/neon/pull/4654/commits/4346b1445740619131c43521e740fa9ad59c1ff0